source/develop/release-management/features/network/copy-host-networking.html.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Copy Host Networking
+category: feature
+authors: Ales Musil
+feature_name: Copy Host Networking
+feature_modules: copy-host-networking
+feature_status: In Development
+---
+
+# Copy Host Networking
+
+## Summary
+
+As it turned out setting up multiple hosts at once can be rather tedious via
+UI. The slow point is that the administrator has to go through every new host
+and do Setup Host Network manually. By utilizing Copy Host Networking, the administrator
+has to set only single host, for the rest the administrator can just specify the host
+to take the network configuration from. 
+
+
+### Owner
+
+- Feature Owner: Ales Musil (amusil)
+- E-mail: <amusil@redhat.com>
+
+
+### Benefit to oVirt
+
+The Copy Host Network helps the administrator with Day 1 deployment automation
+for host networking configuration across multiple hosts. This feature can also be
+on Day 2 for any additional host that needs the same configuration of networking. 
+
+### Example flow
+
+After creation of new Cluster, the administrator adds several hosts (`Host0, Host1, Host2`). 
+`Host0` networking has to be configured manually either by UI or through REST API.
+Other hosts, as long as they have same layout of network interfaces, can copy
+their settings from the host that has been already configured. 
+
+The copy can be done via UI. Going to hosts network interfaces and by clicking
+the `Copy Host Networking` button. In the popup window that will appear 
+the administrator can select from which host (`Host0`) should the configuration be copied.
+
+Via REST API every host has method called `copynetworks` which needs a single
+parameter, id of the host (`Host0`) that the configuration should be copied from. 
+
+### Limitation
+
+* The host configuration can be copied only within the same cluster. 
+
+* Configuration of static IP address for any of the networks cannot be copied. The static IP
+networks have their IP incremented by one to stay in range of the mask. 
+
+* Host with same interface names but different physical network connections will result in bad configuration. 
+See more details in [Implementation](#implementation).
+
+* Destination host has to have same amount or more interfaces than host that we are copying from. In case
+of opposite, we cannot determine if we should create bond with missing interface or skip the bond creation
+and use it only for simple network attachment.
+
+
```

Comment:
What should happen if the destination host has already non-management network configured?

Reply:
As I see it we should replace the configuration with copied one.

Reply:
If you think that this would mean that already configured networks on the destination host should be removed, please write this in the text. Just to avoid misunderstandings about the desired behavior.

Reply:
Done